ios/RNTrackPlayer/Vendor/SwiftAudio/Classes/RemoteCommandController/RemoteCommandController.swift

```diff
@@ -30,9 +30,11 @@ public class RemoteCommandController {
     }
     
     internal func enable(commands: [RemoteCommand]) {
-        self.disable(commands: RemoteCommand.all())
-        commands.forEach { (command) in
-            self.enable(command: command)
+        RemoteCommand.all().forEach { (command) in
```

Comment:
```swift

.skipForward
.skipBackward
.like
.dislike
.bookmark

commands.contains  with these enumeration values may not get the desired results
```

Reply:
Thanks – I will test it with these values.

Reply:
Yeah that didn't work. I added a commit which should fix these cases. A little verbose, but it does the job.